src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql

```diff
@@ -0,0 +1,102 @@
+/* columnar--9.5-1--10.0-1.sql */
+
+CREATE SCHEMA cstore;
+SET search_path TO cstore;
+
+CREATE FUNCTION cstore_fdw_handler()
+RETURNS fdw_handler
+AS 'MODULE_PATHNAME'
+LANGUAGE C STRICT;
+
+CREATE FUNCTION cstore_fdw_validator(text[], oid)
+RETURNS void
+AS 'MODULE_PATHNAME'
+LANGUAGE C STRICT;
+
+CREATE FOREIGN DATA WRAPPER cstore_fdw
+HANDLER cstore_fdw_handler
+VALIDATOR cstore_fdw_validator;
+
+CREATE FUNCTION cstore_ddl_event_end_trigger()
+RETURNS event_trigger
+AS 'MODULE_PATHNAME'
+LANGUAGE C STRICT;
+
+CREATE EVENT TRIGGER cstore_ddl_event_end
+ON ddl_command_end
+EXECUTE PROCEDURE cstore_ddl_event_end_trigger();
+
+CREATE FUNCTION pg_catalog.cstore_table_size(relation regclass)
+RETURNS bigint
+AS 'MODULE_PATHNAME'
+LANGUAGE C STRICT;
+
+CREATE TABLE cstore_data_files (
```

Comment:
probably no need to debate naming here, but it's a little odd that these are prefixed twice (cstore.cstore_data_files)

Reply:
Fair point, not going to change in the merge. This is verbatim copied from the cstore2 repo.

We will need to go through everything named `cstore` anyway and rename to `columnar` after the merge. I think it is a good moment to remove the stutter naming here.